config/default.go

```diff
@@ -204,6 +205,7 @@ CheckStatusCertificateInterval = "5m"
 RetryCertAfterInError = false
 GlobalExitRootL2 = "{{L2Config.GlobalExitRootAddr}}"
 SovereignRollupAddr = "{{L1Config.polygonZkEVMAddress}}"
+AgglayerBridgeL2Addr = "{{L2Config.BridgeAddr}}"
```

Comment:
Is it enough just to call it `BridgeL2Addr`?

```suggestion
BridgeL2Addr = "{{L2Config.BridgeAddr}}"
```

Reply:
we were everywhere using AgglayerBridgeL2, so i thought its better to use this way and be more explicit. wdyt?